services/account.rst

```diff
@@ -120,3 +123,13 @@ links.
   and find 'change password' link. 
   If you cannot find, you are under a single sign on. 
 * `Sumire-PFS slack account settings <https://sumire-pfs.slack.com/account/settings#password>`_
+
+If you have issues on 2FA 
```

Comment:
This paragraph seems not related to **"Lost Account or Change password"** Section.
How does it sound to 
- Add "Help" section, and
- Have "lost account or change password" and "2FA" subsections?